.circleci/config.yml

```diff
@@ -1,320 +0,0 @@
-version: 2.1
```

Comment:
We need to disconnect CircleCI from this repo, and also update the expected checks